src/platform/user/profile/vet360/containers/VAPProfileField.jsx

```diff
@@ -265,29 +280,35 @@ class VAPProfileField extends React.Component {
             className={classes.editButton}
           />
         )}
-      </div>
+      </div>,
     );
 
     if (isEmpty) {
-      content = (
+      content = wrapInTransaction(
         <button
           type="button"
           onClick={this.onAdd}
           className="va-button-link va-profile-btn"
           id={`${this.props.fieldName}-edit-link`}
         >
           Please add your {title.toLowerCase()}
-        </button>
+        </button>,
       );
     }
 
     if (showEditView) {
-      content = <EditView {...childProps} />;
+      content = (
+        <EditView
+          refreshTransaction={this.refreshTransaction}
```

Comment:
Now that the `Vet360Transaction` is no longer rendered when the edit view is, the edit view is responsible for refreshing its own transaction.